README.rst

```diff
@@ -53,6 +53,27 @@ uvloop with::
 Using uvloop
 ------------
 
+Starting with Python 3.12, the preferred way of using uvloop
```

Comment:
uvloop.run seems to be available on all supported Python versions so I think this documentation should be "as of uvloop x.y, the preferred way of..."

Reply:
good one, thanks